390/Serena_Li.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "MP2 Viz"
+author: "Serena Li"
+date: "9/15/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(dplyr)
+library(readr)
+```
+
+```{r}
+stem <- read.csv("scbi.stem1.csv")
```

Comment:
Hey Serena, here is a code review!

A heads up that this file didn't originally knit b/c of the file path issue. i.e. R doesn't know where this CSV is. This is something you need to be careful of for reproducibility. Check out a similar comment I made on Lauren's PR: https://github.com/390-2020-09-Fall/SCBI-ForestGEO-Data/pull/2#discussion_r491179003